src/main/java/com/github/flaxsearch/api/BKDNode.java

```diff
@@ -63,24 +65,31 @@ public void setParent(BKDNode parent) {
         parent.children.add(this);
     }
 
-    public BKDNode findParent(BKDNode node, int numDims, int bytesPerDim) {
-        // go up the tree until we find a node whose parent
-        // contains this node
+    public BKDNode findParentOf(BKDNode node, int numDims, int bytesPerDim) {
+        // go up the tree until we find a node which contains the supplied node
         BKDNode parent = this;
-        while (parent.contains(node, numDims, bytesPerDim) == false) {
+        while (! parent.contains(node, numDims, bytesPerDim)) {
             parent = parent.parent;
```

Comment:
Can we not change these?  I know IntelliJ does this, but I think it makes things a lot less clear (! is easy to miss, '== false' isn't)